public/sass/pages/_explore.scss

```diff
@@ -55,6 +55,25 @@
     margin-top: 2 * $panel-margin;
   }
 
+  .time-series-disclaimer {
+    width: 300px;
+    margin: 10px auto;
+    padding: 10px 0;
+    border-radius: 4px;
+    text-align: center;
+    background-color: #212124;
```

Comment:
should use variables (both for consistency and to support both dark and light themes)